pulsar-broker/src/test/java/org/apache/pulsar/broker/transaction/TransactionTest.java

```diff
@@ -231,7 +232,7 @@ public void testCreateTransactionSystemTopic() throws Exception {
 
         // getList does not include transaction system topic
         List<String> list = admin.topics().getList(NAMESPACE1);
-        assertEquals(list.size(), 2);
+        assertEquals(list.size(), 1);
```

Comment:
Seems we don't care how many topics here, suggestion `assertFalse(list.isEmpty())`